React/tic-tac/src/redux/form/reducer.js

```diff
@@ -0,0 +1,3 @@
+import { reducer } from 'redux-form';
+
+export default reducer;
```

Comment:
This file isn't necessary. I'd delete it and import this reducer directly in `store.js` file